crates/holochain_types/src/app.rs

```diff
@@ -65,6 +65,7 @@ pub struct RegisterDnaPayload {
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub struct NetworkInfoRequestPayload {
     /// The calling agent
+    // TODO should this be restricted to the agent for the current app?
```

Comment:
This is an interesting one. Most things on the app interface are now app restricted, but this isn't. Any agent key you know about can be used here. In theory you don't know who else is local but you could collect agent keys in an app and try them. Seems like we should restrict this to 'current app' too?

Reply:
Yes, I think so, since this is an app api request.